bin/create-action.js

```
@@ -0,0 +1,169 @@
+        choices: colors
+      }
+    ])
+  } catch (err) {
+    // When prompt() rejects, that means the user has pressed ctrl+c to cancel input.
+    throw new Error('Cancelled. Maybe next time!')
+  }
+}
+
+/**
+ * Creates a Dockerfile contents string, replacing variables in the Dockerfile template
+ * with values passed in by the user from the CLI prompt.
+ *
+ * @param {PromptAnswers} answers The CLI prompt answers.
+ * @returns {Promise<string>} The Dockerfile contents.
+ */
+const createDockerfile = async (answers) => {
+  const dockerfileTemplate = await readTemplate('Dockerfile')
+  return dockerfileTemplate
+    .replace(':NAME', answers.name)
+    .replace(':DESCRIPTION', answers.description)
+    .replace(':ICON', answers.icon)
+    .replace(':COLOR', answers.color)
+}
+
+/**
+ * Creates a `package.json` object with the latest version
+ * of `actions-toolkit` ready to be installed.
+ *
+ * @param {string} name The action package name.
+ * @returns {object} The `package.json` contents.
+ */
+const createPackageJson = (name) => {
+  const { version } = require('../package.json')
+  return {
+    name,
+    private: true,
+    main: 'index.js',
+    dependencies: {
+      'actions-toolkit': `^${version}`
+    }
+  }
+}
+
+/**
+ * Runs the create action CLI prompt and bootstraps a new directory for the user.
+ *
+ * @public
+ * @param {string[]} argv The command line arguments to parse.
+ * @returns {Promise<void>} Nothing.
+ */
+const createAction = async (argv) => {
+  const args = minimist(argv)
+  const directoryName = args._[0]
+  if (!directoryName || args.help) {
+    console.log(`\nUsage: npx actions-toolkit <name>`)
+    process.exit(1)
+    // Although this return is unreachable,
+    // for some reason, code after this block is reached in unit tests,
+    // even while this calls `process.exit(1)`.
+    // Adding a `return` below fixes that issue in the tests.
+    // eslint-disable-next-line
+    return
```

Check out the comments above - just a heads up.

Another approach I can think of is for this function not to call `process.exit()` at all - Maybe just throw an Error here and let the `.catch()` in `bin/cli.js` print the error message (`\nUsage: npx actions-toolkit <name>`). 

Thoughts either way? Something else in mind here?

---

How about this:

```suggestion
    return process.exit(1)
```

I'm not actually sure if this does what I think it does, but it'd avoid the "functionally unnecessary" extra `return` bit 🤔

---

This will both work for tests and the CLI help / directory name argument functionality. Good call - will update.